Add-ons/UmbracoForms/Developer/Working-With-Data/index-v8.md

```diff
@@ -6,10 +6,13 @@ meta.Description: "Developer documentation on working with Forms record data."
 
 # Working with Record data
 
-From Umbraco Forms `v8.2` includes some helper methods that return records of a given form, which can be used to output records in your templates using razor.
+Umbraco Forms includes some helper methods that return records of a given form, which can be used to output records in your templates using Razor.
 
-## Available methods
-The methods can be found by injecting the `Umbraco.Forms.Core.Services.IRecordReaderService` interface. For performance reasons, all these methods are paged.
+## Working with records in bulk
+
+The methods can be found by injecting the `Umbraco.Forms.Core.Services.IRecordReaderService` interface.
+
+For performance reasons, all these methods are paged and retrieve their values from an underying Examine search index.
```

Comment:
underlying

Reply:
Thanks - have updated this and the other corrections.